srcpkgs/drone-runner-exec/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'drone-runner-exec'
+pkgname=drone-runner-exec
+version=1.0.0
+revision=1
+build_style=go
+go_import_path=github.com/drone-runners/drone-runner-exec
+go_package="${go_import_path}"
+short_desc="Exec Runner for Drone continuous delivery platform"
+maintainer="shanduur <murbanek@shanduur.com>"
+license="custom:Polyform-Small-Business-1.0.0, custom:Polyform-Free-Trial-1.0.0"
```

Comment:
haven't looked at the licences themselves but this will probably be `repository=nonfree` and possibly `restricted=yes`

Reply:
I added both, but I am not sure about the restricted. I mean - it is free for personal use and for Small Business defined as:

```
Use of the software for the benefit of your company is use for
a permitted purpose if your company has fewer than 100 total
individuals working as employees and independent contractors,
and less than 1,000,000 USD (2019) total revenue in the prior
tax year.  Adjust this revenue threshold for inflation according
to the United States Bureau of Labor Statistics' consumer price
index for all urban consumers, U.S. city average, for all items,
not seasonally adjusted, with 1982-1984=100 reference base.
```

And provides Free Trial for large businesses:

```
Use to evaluate whether the software suits a particular
application for less than 32 consecutive calendar days, on
behalf of you or your company, is use for a permitted purpose.
```

This probably means it could be placed in the unrestricted repo?